server/test/services/fights.service.spec.ts

```diff
@@ -272,4 +283,122 @@ describe('FightsService', () => {
       );
     });
   });
+
+  describe('newEvents', () => {
+    let events: Event[];
+    const redPlayerPoints = 2;
+    const bluePlayerPoints = 1;
+
+    beforeEach(() => {
+      fight.timer = new Timer(1);
```

Comment:
wydaje mi się, że nie ma takiej potrzeby, żeby tutaj ustawiać nowy timer i dodawać przy każdym teście na nowo sędziów i startować walkę, wystarczy beforeAll ustawiający listę events tak jak tutaj. Wtedy należy jeszcze usunąć niepotrzebne ustawianie state walki na Scheduled w 'should not add events for scheduled fight' i ustawić state na Running w 'should add new events for finished fight'